FMI-CROSS-CHECK-RULES.md

```diff
@@ -274,17 +244,6 @@ Placeholder   | Possible Values
 ```
 
 
```

Comment:
If we drop 9.2.5 I would add something like
"Upon request, importers must present simulation results and a comparison to the reference solution provided by the exporter of the FMU to the FMI Steering Committee, which may reject results that differ significantly from the reference."

Reply:
Excellent proposal, added as is.